extension/scripts/content/city/ttCity.js

```diff
@@ -0,0 +1,189 @@
+"use strict";
+
+(async () => {
+    await loadDatabase();
+    if (!settings.pages.city.highlight && !settings.pages.city.showValue) return;
```

Comment:
This shouldn't be there. v6 will have live-settings, having this would prevent that from working.